doc/classes/AudioStream.xml

```diff
@@ -16,31 +16,39 @@
 		<method name="_get_beat_count" qualifiers="virtual const">
 			<return type="int" />
 			<description>
+				Should return the total number of beats of this audio stream. Used by the engine to determine the position of every beat.
```

Comment:
I think we should always explicitly tell the user that virtual methods need to be implemented by them. Without this clarification a description starting with "Should return" looks concerning, as if we don't know exactly if this method returns what we think it returns.

Same below.

Reply:
The usual "_Override this method to[...]_" cannot really work here otherwise it would be too verbose.

I would introduce them with "_Overridable method._" personally. I don't like to say "_Virtual method to be implemented by the user_" because it feels very redundant, but that's obviously just an opinion. `virtual` is right there, you can see what it does, and all virtual methods begin with an underscore. Users should be used by this, especially if they're trying to make their own audio streams.